public/js/components/RightSidebar.js

```diff
@@ -80,6 +80,11 @@ const RightSidebar = React.createClass({
 
   setupKeyboardShortcuts() {
     const { keyShortcuts } = this.props;
+    if (this.keyShortcutsEnabled) {
+      return;
+    }
+
+    this.keyShortcutsEnabled = true;
```

Comment:
Why not just check `if(this.keyShortcuts) { ... }` ?

You're passing down a prop that is created in the parent's `componentDidMount`, so it makes sense to be null when it's rendering first.

There's probably a better way to do this, most likely using context to expose shortcuts across the whole app, or even just using a top-level object. Not sure.